cssResponsive/css-responsive_responsive-website/css/styles.css

```diff
@@ -243,11 +244,32 @@ CSS when certain conditions that we declare are met - such as a certain viewport
 width, etc. */
 
 /* Media Query Tablet */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
-
-/* Media Query Desktop */
-/* --- */
-/* Replace this comment with your media query. */
-/* --- */
+@media screen and (min-width: 600px) {
+  .card-section {
+    display: flex;
+    justify-content: center;
+    flex-flow: wrap;
+    gap: var(--gap);
+  }
+
+  .product-card {
+    flex-wrap: wrap;
+    width: 40%;
```

Comment:
Why `40%`?

Reply:
trial and error to see what fits best and looks closest to the preview